doc/source/misc.rst

```diff
@@ -99,3 +149,11 @@ Sometimes it is useful to monitory the memory footprint of the python process.
 This can be achieved by using ``memory_profiler``
 (https://pypi.python.org/pypi/memory_profiler).
 
+
+CPU profiling of C extensions
```

Comment:
Testing related stuff fits better in the developer.rst in my oppinion.

Reply:
I moved it to developer.rst